typescript/index.d.ts

```diff
@@ -205,7 +205,7 @@ export function isRecoilValue(val: unknown): val is RecoilValue<any>; // eslint-
 // bigint not supported yet
 type Primitive = undefined | null | boolean | number | symbol | string;
 
-export type SerializableParam = Primitive | SerializableParam[] | { [key: string]: SerializableParam };
+export type SerializableParam = Primitive | ReadonlyArray<SerializableParam> | { [key: string]: SerializableParam };
```

Comment:
Read only object as well?

Reply:
I think it's okay as is. That can receive any objects which has readonly properties. 👌